libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -240,6 +243,8 @@ public void run() {
             }, t.getDuration());
             t.show();
         }
+        final Intent intent = new Intent(AUTHENTICATION_FAILED_INTENT);
```

Comment:
I was able to receive the intent! Just wanted to confirm before I report back to UX and PM, there's no way to differentiate the errors?